release/release.go

```diff
@@ -55,16 +55,8 @@ var (
 // ForceDevMode returns true if the distribution doesn't implement required
 // security features for confinement and devmode is forced.
 func (o *OS) ForceDevMode() bool {
-	for _, req := range requiredApparmorFeatures {
-		// Also ensure appamor is enabled (cannot use
-		// osutil.FileExists() here because of cyclic imports)
-		p := filepath.Join(apparmorFeaturesSysPath, req)
-		if _, err := os.Stat(p); err != nil {
-			return true
-		}
-	}
-
-	return false
+	level, _ := apparmor.Probe()
```

Comment:
I'd like to see a prominent (ie,non-DEBUG) log message stating that snapd is going to force all snaps to be in devmode, with the reason why. Because you discard the error here, we can't do that.

Are you planning on doing this in a follow-up PR?

Reply:
I have this already in the branch that uses this. I think that once we get to the warnings framework it will be a proper user-visible message.